src/test/java/lotto/domain/WinLottoNumberTest.java

```diff
@@ -11,18 +13,14 @@
 class WinLottoNumberTest {
 
     @Test
-    void 당첨_번호와_보너스번호_중복확인() {
-        LottoNumbers lottoNumber = new LottoNumbers(Arrays.asList(1, 3, 5, 7, 9, 11));
-        int bonusNumber = 5;
-        assertThat(new WinLottoNumber(lottoNumber, bonusNumber));
-    }
+    void 당첨_번호와_보너스번호_중복이_불가하다() {
 
-    @Test
-    void 당첨_결과를_정상_확인() {
-        LottoNumbers resultLottoNumber = new LottoNumbers(Arrays.asList(1, 3, 5, 7, 9, 11));
-        LottoNumbers requestLottoNumber = new LottoNumbers(Arrays.asList(1, 3, 5, 2, 4, 6));
-        int bounusNumber = 13;
-        WinLottoNumber winLottoNumber = new WinLottoNumber(resultLottoNumber, bounusNumber);
-        assertThat(winLottoNumber.matchingLottoNumber(requestLottoNumber)).isEqualTo(3);
+        Set<LottoNumber> lottoNumber = new HashSet<>();
+        for(int i : Arrays.asList(1, 3, 5, 7, 9, 11))
+        {
+            lottoNumber.add(new LottoNumber(i));
+        }
```

Comment:
```suggestion
        for(int i : Arrays.asList(1, 3, 5, 7, 9, 11)) {
            lottoNumber.add(new LottoNumber(i));
        }
```

```java

    Set<LottoNumber> lottoNumber = Set.of(
        new LottoNumber(1),
        new LottoNumber(3),
        new LottoNumber(5),
        new LottoNumber(7),
        new LottoNumber(9),
        new LottoNumber(11)
    );
```
이렇게 사용도 가능하겠네요. 

`WinLottoNumber#matchingLottoNumber` 에 대한 테스트도 추가되면 좋을 것 같아요.